projects/aca-content/src/lib/store/effects/node.effects.ts

```diff
@@ -460,4 +461,26 @@ export class NodeEffects {
       ),
     { dispatch: false }
   );
+
+  folderInformation$ = createEffect(
```

Comment:
could you please write some unit tests for this effect?

Reply:
Added unit test